linear_algebra/code/CMakeLists.txt

```diff
@@ -48,30 +48,15 @@ include_directories(include test .)
 
 add_executable(la_test ${Sources} ${Includes})
 
-set(CMAKE_VERBOSE_MAKEFILE 1)
-
-if(CXX_COMPILER STREQUAL clang++)
-    set(CMAKE_C_COMPILER   clang)
-    set(CMAKE_CXX_COMPILER clang++)
-else()
-    set(CMAKE_C_COMPILER   gcc)
-    set(CMAKE_CXX_COMPILER g++)
-endif()
-
-if(CMAKE_CXX_COMPILER STREQUAL clang++)
-
-    set(CMAKE_C_FLAGS_DEBUG   "-g -Wall -pedantic -Wextra")
-    set(CMAKE_CXX_FLAGS_DEBUG "-g -std=c++17 -Wall -pedantic -Wextra -Wno-unused-parameter -Wno-unused-function")
```

Comment:
No need to set "-Wno-unused-function" manually, as setting "-Wall" sets these flags automatically: https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html

Reply:
As discussed, my mistake, this was the "no" variant of the flag and is now added back.  Ignore comment